mathesar/models/base.py

```diff
@@ -52,6 +52,7 @@
 from mathesar.state import make_sure_initial_reflection_happened, get_cached_metadata, reset_reflection
 from mathesar.state.cached_property import cached_property
 from mathesar.api.exceptions.database_exceptions.base_exceptions import ProgrammingAPIException
+from django.contrib.postgres.fields import ArrayField
```

Comment:
We've decided to support `sqlite` for our internal database going forward. The `ArrayField` is a postgres specific field and isn't compatible with sqlite. Therefore, you should stick with `JSONField` for `column_order`.